supporting_content/dimensions/region.md

```diff
@@ -0,0 +1,69 @@
+|-----------|----------------------------|------------|--------------------|---------------------------------------|-----------------|-----------------------|
+| AWS       | eu-west-1                  | eu-west-1a | single-region      | EU?                                   | Ireland?        | GMT +/- n             |
+| GCP       | europe-west3               |            | single-region      |                                       |                 |                       |
+| Microsoft | WestEurope                 |            | single-region      |                                       | Ireland         |                       |
+| AWS       | Global (normalized?)       |            | global             |                                       |                 |                       |
+| GCP       | nam-eur-asia1              |            | multi-region       |                                       |                 |                       |
+
+
+| Provider | Summary (lowest level) | Geo Region (code? Continent) | Country (code?) | Provider Region / Location | AZ / Zone  | Region Category(?) | TimeZone (check need) |
+|----------|------------------------|------------------------------|-----------------|----------------------------|------------|--------------------|-----------------------|
+| AWS      | eu-west-1a             | EU?                          | Ireland?        | eu-west-1                  | eu-west-1a | single-region      | GMT +/- n             |
+| GCP      | europe-west3           | EU                           | Ireland?        | europe-west3               | (null)     | single-region      |                       |
+| MS       | WestEurope             | EU                           | Ireland         | WestEurope                 | (null)     | single-region      |                       |
+| AWS      | Global                 | Global                       | Global          | Global (normalized?)       | (null)     | global             |                       |
+| GCP      | nam-eur-asia1          | Global?                      | Global?         | nam-eur-asia1              | (null)     | multi-region       |                       |
+
+-   Brainstorming secondary region
+| Provider | Provider Region | Secondary Region | Secondary Region Type   |
+|----------|-----------------|------------------|-------------------------|
+|          | WestEurope      | BrazilSouth      | DataTransferDestination |
+|          | WestEurope      | External         | DataTransferSource      |
+|          | WestEurope      |                  |                         |
+
+- References:
+  - AWS: <https://docs.aws.amazon.com/cur/latest/userguide/product-columns.html#R>
+  - GCP: <https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage#detailed-usage-cost-data-schema>
```

Comment:
We should link to STANDARD export vs. DETAILED export - 
https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/standard-usage